bHY6yh-d30103/cod-bqe7s7_bHY6yh-d30103.xml

```diff
@@ -0,0 +1,491 @@
+               <lb ed="#P"/>est qui eorum que putat sua esse illi opera<lb ed="#P" break="no"/>tuis 
+               est proptur illud.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e355">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e392">Ibidme libro ii capitulo iiii</ref> <quote xml:id="bHY6yh-d30103-Qd1e357">amici condolent</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e363">
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e372">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e419">Ibidem capitulo vi</ref> <quote>amare est velle alicui 
+               <lb ed="#P"/>qua putat bona illius sed non sui et secundum 
+               <lb ed="#P"/>posse actiuum esse horum. amicus est qui 
+               <lb ed="#P"/>amat / et econtra amatur. putantur autem ami<lb ed="#P" break="no"/>ci 
+               esse qui sic habere putantur adiuicem
+               <lb ed="#P"/>Nec esse est amicum esse congaudentem 
+               <lb ed="#P"/>bonis / et condolentem tristibus / Qui ea 
+               <lb ed="#P"/>qui sibi vult / huic videtur amicus esse
+               <lb ed="#P"/>Factura amicitie est gratia / et non re<lb ed="#P" break="no"/>quisitum 
+               facere et cum fecerit non 
+               <lb ed="#P"/>manifestare: sic enim impsius gratia 
+               <lb ed="#P"/>videtur non proptur alterum.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e397">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e442">Ibidem xvi capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e399">iuuenes sunt amatores
+               <lb ed="#P"/>Amicitie et amici / et amatores so<lb ed="#P" break="no"/>dalium 
+               magis quam aliarum etatum quia ga<lb ed="#P" break="no"/>udent 
+               in conueniendo.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e407">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e454">Idem politicorum libro ii capitulo primo</ref> <quote xml:id="bHY6yh-d30103-Qd1e409">amici<lb ed="#P" break="no"/>tiam 
+               quidem putamus maximum esse bo<lb ed="#P" break="no"/>norum 
+               ciuitatibus / sic enim utique minime 
+               <pb ed="#P" n="59-v"/>
+               <cb ed="#P" n="a"/>
+               <lb ed="#P"/>seditiones faciet.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e419">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e469">Idem topicorum libro iii</ref> <quote xml:id="bHY6yh-d30103-Qd1e421">Magis eligenda est 
+               <lb ed="#P"/>super abundantia amicorum quam pecuniarum</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e433">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e487">Cronica de nugis philosophorum capitulo xxxiii ¶</ref> <quote xml:id="bHY6yh-d30103-Qd1e437">de 
```

Comment:
check the pilcrow placement here. I don't think it should be inside the `<ref>` or inside the `<quote>` 

Rather it should come between them.

See below.

```suggestion
               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e487">Cronica de nugis philosophorum capitulo xxxiii</ref> ¶ <quote xml:id="bHY6yh-d30103-Qd1e437">de 
```